app/src/main/res/values/strings.xml

```diff
@@ -384,4 +384,8 @@
     <string name="password_validation_identical_characters">No more than 2 identical characters in a row\n</string>
     <string name="password_validation_minimum_characters">Use a minimum of 8 characters\n</string>
     <string name="password_validation_one_number_one_letter">Password with at least 1 number and 1 letter\n</string>
+    <string name="terms">Terms</string>
+    <string name="terms_info"><![CDATA[By continuing you accept and agree with the <a href="https://gnosis.pm">Terms of Use</a> and <a href="https://gnosis.pm/resources/default/pdf/PrivacyPolicyGnosisLtd.pdf">Privacy Policy</a>]]></string>
+    <string name="no_thanks">NO THANKS</string>
```

Comment:
I normally use all caps in the textview, so I can reuse it if necessary ... but doesn't really matter :D